assets/components/service-ca/deployment.yaml

```diff
@@ -23,45 +22,45 @@ spec:
         app: service-ca
         service-ca: "true"
     spec:
-      serviceAccount: service-ca
       serviceAccountName: service-ca
       containers:
-      - name: service-ca-controller
-        image: {{ .ReleaseImage.service_ca_operator }}
-        imagePullPolicy: IfNotPresent
-        command: ["service-ca-operator", "controller"]
-        ports:
-        - containerPort: 8443
-        securityContext:
-          runAsNonRoot: true
-        resources:
-          requests:
-            memory: 120Mi
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /var/run/secrets/signing-key
-          name: signing-key
-        - mountPath: /var/run/configmaps/signing-cabundle
-          name: signing-cabundle
+        - name: service-ca-controller
```

Comment:
this whitespace is not present in https://github.com/openshift/service-ca-operator/blob/master/bindata/v4.0.0/controller/deployment.yaml

Reply:
The whitespace change is the result from `yq`'s reformatting.